explore_utils.py

```diff
@@ -101,3 +101,21 @@ def find_channel_grouping_revenue(dataset):
     means = {grouping: np.mean(df[df['channelGrouping'] == grouping]['totals.transactionRevenue'].astype('int64')) / 10000 for grouping in groupings}
 
     return counts, means
+
+def find_transaction_by_region(data):
+    """ Find the average transaction revenue by region
+
+    args:
+        dataset (Dataset): the google analytics Dataset
+
+    returns:
+        Dataframe of total transaction revenue by region in ascending order.
+
+    """
+    train_df = data.train
+    train_df['rev'] = train_df['totals.transactionRevenue'].fillna(0).astype(float)
```

Comment:
This adds the 'rev' column to `train_df`. Since `train_df` refers to the same object as `data.train`, this change is then reflected back in `explore.py` for everyone. I added some print statements to `explore.py` to show this:

Here is the code:
```
    print('rev before?', 'rev' in data.train.columns)
    trans_by_region = explore_utils.find_transaction_by_region(data)
    print("The transaction revenues by region are: ", trans_by_region)
    print('rev after?', 'rev' in data.train.columns)
```

And here is the output:
```
rev before? False
The transaction revenues by region are:                                  Transaction
geoNetwork.region                          
not available in demo dataset  1.400458e+06
Washington                     3.440000e+06
California                     6.227701e+06
Michigan                       6.310000e+06
New York                       1.218167e+07
rev after? True
```

Note how `rev` is in the dataframe after calling your function. Generally we don't want to do this, so please a shallow copy to create a new dataframe. (More info: https://pandas.pydata.org/pandas-docs/stable/generated/pandas.DataFrame.copy.html)